src/test/java/com/aws/greengrass/clientdevices/auth/session/MqttSessionFactoryTest.java

```diff
@@ -107,4 +119,31 @@ void GIVEN_componentWithValidClientId_WHEN_createSession_THEN_returnsSession() t
         assertThat(session, is(IsNull.notNullValue()));
         assertThat(session.getSessionAttribute(Component.NAMESPACE, "component"), notNullValue());
     }
+
+    @Test
+    void GIVEN_credentialsWithCertificateBeyondTrustDuration_WHEN_createSession_THEN_throwsAuthenticationException()
+            throws InvalidCertificateException {
+        Certificate activeCert = CertificateFake.activeCertificate("fake-cert", Instant.EPOCH);
+        when(mockCertificateRegistry.getCertificateFromPem(any()))
```

Comment:
I am not a big fan of mocking when we could avoid it (generally I only prefer mocks for things that our outside of out control like an external lib).  Do we really need to be mocking the Certificate, creating a fake? We already have tooling for testing that creates certificates for us to test with.

This is (optional) so don't pay too much attention to it but just a thought